spec/fluent.ebnf

```diff
@@ -54,24 +57,30 @@ text                 ::= text-char+
 /* in quoted-text " must be escaped */
 quoted-text          ::= '"' (text-char - '"' | '\"')+ '"'
 
-
 placeable            ::= '{' __? (inline-expression | block-expression) __? '}'
-
 inline-expression    ::= quoted-text
                        | number
-                       | identifier
-                       | external
-                       | attribute-expression
-                       | variant-expression
+                       | external-identifier
+                       | public-identifier
+                       | public-attribute-expression
+                       | private-identifier
+                       | private-variant-expression
                        | call-expression
                        | placeable
 block-expression     ::= select-expression
                        | variant-list
 
-select-expression    ::= inline-expression __ '->' __ variant-list
-attribute-expression ::= identifier '.' identifier
-variant-expression   ::= identifier '[' _? variant-key _? ']'
+select-expression    ::= selector-expression __ '->' __ variant-list
+selector-expression  ::= quoted-text
+                       | number
+                       | external-identifier
+                       | public-identifier
```

Comment:
This should be private-identifier instead of public-identifier, as tags are only on private messages, and I guess the intent is to not select based on translation value?

Reply:
Good catch and yes, that's the intent. At least for now.